src/messaging/dotnet/src/Client/Client/WebSocket/WebSocketEnvironmentVariables.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Morgan Stanley makes this available to you under the Apache License,
+// Version 2.0 (the "License"). You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0.
+// 
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership. Unless required by applicable law or agreed
+// to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions
+// and limitations under the License.
+
+namespace MorganStanley.ComposeUI.Messaging.Client.WebSocket;
+
+public static class WebSocketEnvironmentVariables
+{
+    public const string UriEnvironmentVariableName = "COMPOSEUI_MESSAGEROUTER_WEBSOCKET_URI";
```

Comment:
Nitpick: since the class is called `WebSocketEnvironmentVariables` the `EnvironmentVariable` is redundant in `UriEnvironmentVariableName`.